rpcserver.go

```diff
@@ -519,3 +521,53 @@ func (r *rpcServer) ShowRoutingTable(ctx context.Context,
 		Channels: channels,
 	}, nil
 }
+
+func (r *rpcServer) SendMultihopPayment(ctx context.Context, in *lnrpc.MultihopPaymentRequest) (* lnrpc.MultihopPaymentResponse, error){
+	pathArr := make([][32]byte, 0)
+	for i:=0; i<len(in.LightningIDs); i++ {
+		addr, err := wire.NewShaHash([]byte(in.LightningIDs[i]))
+		if err != nil {
+			return nil, err
+		}
+		pathArr = append(pathArr, [32]byte(*addr))
+	}
+	r.server.paymentManager.SendPayment(lnwire.CreditsAmount(in.Amount), pathArr)
+	rpcsLog.Infof("[SendMultihopPayment] amount=%v path=%v", in.Amount, pathArr)
+	return &lnrpc.MultihopPaymentResponse{}, nil
+}
+
+func (r *rpcServer) FindPath(ctx context.Context, in *lnrpc.FindPathRequest) (*lnrpc.FindPathResponse, error){
+	rpcsLog.Info("[FindPath] dest=", hex.EncodeToString([]byte(in.TargetID)))
+	paths, err := r.server.routingMgr.FindKSP(
+		graph.NewID(in.TargetID),
+		int(in.NumberOfPaths),
+		1*time.Second,
+	)
+	rpcsLog.Info("[FindPath] after FindKSP dest=", hex.EncodeToString([]byte(in.TargetID)))
```

Comment:
Converting to a byte slice, then using `EncodeToString` seems like it'll lead to an erroneous string being produced. Why not just encode as hex directly? Or if it's already a hex string, then that can be printed as is.